### PR TITLE
Lib fork

### PR DESCRIPTION
**Pull request by StrixLuca**
It functions

i adapted it for qbx lib

But that bug
that you have to restart the script in game for it to work

It's still there

### CODE REVIEW

**Review by solareon — CHANGES_REQUESTED** [review]
Please convert ar/cs/es/fr since we had them before.

File: client/main.lua [hunk 1]
```diff
@@ -268,16 +268,12 @@ local function destroyInsideZones()
     destroyDeliveryTarget()
 end
 
-local function loadAnimDict(dict)
-    while (not HasAnimDictLoaded(dict)) do
-        RequestAnimDict(dict)
-        Wait(5)
-    end
-end
+
```
Comment (BerkieBb) [inline client/main.lua:271]: so much empty space which doesn't need to be here

File: fxmanifest.lua [hunk 2]
```diff
@@ -4,12 +4,11 @@ game 'gta5'
...
 version '2.1.0'
+ox_lib 'locale'
 
-shared_script {
+shared_scripts {
     '@ox_lib/init.lua',
-    '@qbx_core/shared/locale.lua',
-    'locales/en.lua',
-    'locales/*.lua',
+   '@qbx_core/modules/lib.lua',
```
Comment (BerkieBb) [inline fxmanifest.lua:11]: indentation is off here

File: fxmanifest.lua [hunk 3]
```diff
@@ -22,7 +21,7 @@ client_scripts {
 
 files {
   'config/client.lua',
+  'locales/*.json',
 }
-
```
Comment (BerkieBb) [inline fxmanifest.lua:26]: put this space back

**Review by solareon — APPROVED** [review]

**Review by BerkieBb — APPROVED** [review]

**Pull request merged**